mnelab/model.py

```diff
@@ -443,6 +443,31 @@ def epoch_data(self, events, tmin, tmax, baseline):
         self.current["events"] = self.current["data"].events
         self.current["name"] += " (epoched)"
 
+    @data_changed
+    def convert_od(self):
+        self.current["data"] = mne.preprocessing.nirs.optical_density(
+            self.current["data"])
+        self.current["name"] += f" (OD)"
+        self.history.append(
+            f'data = mne.preprocessing.nirs.optical_density(data)')
+
+    @data_changed
+    def apply_tddr(self):
+        self.current["data"] = mne.preprocessing.nirs.\
+            temporal_derivative_distribution_repair(self.current["data"])
```

Comment:
Maybe import this function with a shorter name? E.g. `tddr`?

Reply:
If I make it `tddr` this will still cross two lines. Plus there will be the added line for the import. So this is less lines in total and is more explicit. But happy to add the import, do you prefer all imports up the top, or should I nest the import in `apply_tddr`?

Reply:
If you import the function as `from mne.preprocessing.nirs import temporal_derivative_distribution_repair as tddr` up top this will fit into one line.

I'd not do this for the history entry though, because fiddling with imports in the history will complicate things. Therefore, the history entry is fine as it is.

However, would it make sense to rename this function to `tddr` directly in MNE? We often use abbreviations, and given that this function is in the `nirs` subpackage, it should be clear to NIRS people what this function does (otherwise there's still the docstring).

Reply:
Great I will make the changes as you suggest.

Regarding changing the name in MNE, I orginally submitted the code as a PR with `tddr` naming, but was asked to change it to what it is now. So I think we just leave it.

Reply:
OK. I'm sure that you could convince others that the abbreviation is a much better name, but that's your call. I'm fine with leaving this for now.